src/wmnds/components/message/summary/_summary.njk

```diff
@@ -7,6 +7,9 @@
 {# Success #}
 {% elseif params.type === "success-fill" %}
   {% set iconName = 'general-success' %}
+{# Success #}
+{% elseif params.type === "success-fill-no-icon" %}
```

Comment:
Like this addition, can you add it to the https://deploy-preview-386--wmnetwork.netlify.app/components/message/ page.